Source/Core/Core/HW/ProcessorInterface.cpp

```diff
@@ -111,7 +112,9 @@ void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
 
   mmio->Register(base | PI_FIFO_RESET, MMIO::InvalidRead<u32>(),
                  MMIO::ComplexWrite<u32>([](u32, u32 val) {
-                   WARN_LOG_FMT(PROCESSORINTERFACE, "Fifo reset ({:08x})", val);
+                   INFO_LOG_FMT(PROCESSORINTERFACE, "Wrote PI_FIFO_RESET: {:08x}", val);
+                   if (val != 0)
```

Comment:
probably only a single bit?

Reply:
Changed.  My attempt at hardware-testing found that writing `2` seemed to hang everything, though I also ran into other issues (after calling `GX_AbortFrame` other drawing commands stopped working, but I might be doing something wrong)